Sources/Core/AWSClientRuntime/Middlewares/AWSS3ErrorWith200StatusXMLMiddleware.swift

```diff
@@ -33,6 +33,8 @@ public struct AWSS3ErrorWith200StatusXMLMiddleware<OperationStackOutput>: Middle
             return response
         }
 
+        response.httpResponse.body = .data(data)
+
```

Comment:
This change should have merged in separately with `main`, right?

Reply:
I added this line separately here for now to avoid merging in all the new changes from main that would result in lots of merge conflicts.

Reply:
Just merged all the changes from main, since I have to do that anyways before merging project branch to main branch.